<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>552</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>EncoderGenerator</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="toolTipDuration">
   <number>-3</number>
  </property>
  <property name="styleSheet">
   <string notr="true">QTableWidget{
color:red;
background-color:black;

}
QComboBox{
	background-color: black;
	border: 2px solid red;
	border-radius: 5px;
	color: red;

}
QGroupBox{
    margin-top: 0.5em;
    font: 14px arial;
    color: black;
	border: 3px solid red;
	border-radius: 20px;
}

QGroupBox::title {
	color: rgb(255, 0, 0);
    top: -6px;
    left: 15px;
}
QMenuBar{
	background-color: rgb(252, 221, 255);
}
QMainWindow{
	background-color: rgb(0, 0, 0);
	border: 1px solid red;
}

QPushButton {    
	color: rgb(255, 0, 0);
	background-color:black;
    border-style: outset;
    border-width: 3px;
	border-radius: 5px;
    border-color:red;
}
QLabel { 
	background-color: black;
	color:red;
	border-style: inset ; 
    border-width: 0px;
    border-color: rgb(127, 127, 127); 
	border-radius: 5px}
QDoubleSpinBox{
	background-color: black;
	color:red;
	border: 2px solid gray;
	border-radius: 5px;
	/*background-color: rgb(231, 198, 228);*/
	border-color:rgb(255, 230, 255);
    }

QTabWidget { /* The tab widget frame */
    border-top: 2px solid green;
	background-color:black;
	border: 2px solid red;		
}
QTabBar{
	padding: -16px;
	qproperty-drawBase: 0;
	border: 2px solid red;		
	border-radius: 10px;
	/*background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 0, 0, 255), stop:1 rgba(0, 0, 0, 255));*/
	background-color:black;
	color: red;
	border-style: solid;
}
QTabBar::tab {
	border: 2px solid red;		
	border-radius: 8px;
	background-color: red;
    padding:2px;
}
QTabBar::tab:selected, QTabBar::tab:hover {
	background-color: red;
	color:black;
}
QTabBar::tab:!selected {
    /*margin-top: 4px;  make non-selected tabs look smaller */
	background-color: black;
	color: red;
}

QListWidget {
    color: rgb(255, 0, 0);
	/*border: 3px solid red;*/
	border-radius: 20px;
	background: black;
	selection-background-color: rgb(170, 255, 0);
	selection-color: rgb(255, 255, 0);
}</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>GroupBox</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5" rowstretch="0" columnstretch="0">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>5</number>
           </property>
           <property name="rightMargin">
            <number>7</number>
           </property>
           <property name="bottomMargin">
            <number>5</number>
           </property>
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="spacing">
              <number>7</number>
             </property>
             <item>
              <widget class="QComboBox" name="ComprtCombo">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnSample">
               <property name="text">
                <string>Run test</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lblTime">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Axial</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>5</number>
           </property>
           <property name="rightMargin">
            <number>7</number>
           </property>
           <property name="bottomMargin">
            <number>5</number>
           </property>
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QLabel" name="lblENC2">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>150</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="layoutDirection">
                  <enum>Qt::RightToLeft</enum>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::NoFrame</enum>
                 </property>
                 <property name="text">
                  <string>Pulses per mm    </string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="ddsbxENC2">
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                 <property name="buttonSymbols">
                  <enum>QAbstractSpinBox::NoButtons</enum>
                 </property>
                 <property name="showGroupSeparator" stdset="0">
                  <bool>true</bool>
                 </property>
                 <property name="decimals">
                  <number>6</number>
                 </property>
                 <property name="value">
                  <double>1.740000000000000</double>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnStopAxial">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Stop axial</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QTabWidget" name="tabWidget">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="tabShape">
                <enum>QTabWidget::Rounded</enum>
               </property>
               <property name="currentIndex">
                <number>1</number>
               </property>
               <property name="elideMode">
                <enum>Qt::ElideNone</enum>
               </property>
               <property name="usesScrollButtons">
                <bool>true</bool>
               </property>
               <property name="documentMode">
                <bool>true</bool>
               </property>
               <property name="tabsClosable">
                <bool>false</bool>
               </property>
               <property name="tabBarAutoHide">
                <bool>false</bool>
               </property>
               <widget class="QWidget" name="tab">
                <attribute name="title">
                 <string>Speed</string>
                </attribute>
                <layout class="QGridLayout" name="gridLayout_7">
                 <item row="0" column="0">
                  <widget class="QListWidget" name="lwdgENC2">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="baseSize">
                    <size>
                     <width>0</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <property name="verticalScrollBarPolicy">
                    <enum>Qt::ScrollBarAlwaysOff</enum>
                   </property>
                   <property name="sizeAdjustPolicy">
                    <enum>QAbstractScrollArea::AdjustToContents</enum>
                   </property>
                   <property name="autoScrollMargin">
                    <number>16</number>
                   </property>
                   <property name="alternatingRowColors">
                    <bool>false</bool>
                   </property>
                   <property name="movement">
                    <enum>QListView::Static</enum>
                   </property>
                   <property name="resizeMode">
                    <enum>QListView::Adjust</enum>
                   </property>
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <property name="viewMode">
                    <enum>QListView::ListMode</enum>
                   </property>
                   <property name="uniformItemSizes">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="tab_2">
                <attribute name="title">
                 <string>aux</string>
                </attribute>
                <layout class="QGridLayout" name="gridLayout_8">
                 <item row="0" column="0">
                  <widget class="QTableWidget" name="tableWidget"/>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="title">
           <string>Rotation</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>5</number>
           </property>
           <property name="rightMargin">
            <number>7</number>
           </property>
           <property name="bottomMargin">
            <number>5</number>
           </property>
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QLabel" name="lblENC1">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>150</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="layoutDirection">
                  <enum>Qt::RightToLeft</enum>
                 </property>
                 <property name="text">
                  <string>Pulses per *</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="ddsbxENC1">
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                 <property name="buttonSymbols">
                  <enum>QAbstractSpinBox::NoButtons</enum>
                 </property>
                 <property name="showGroupSeparator" stdset="0">
                  <bool>true</bool>
                 </property>
                 <property name="prefix">
                  <string/>
                 </property>
                 <property name="suffix">
                  <string> pulses per *</string>
                 </property>
                 <property name="decimals">
                  <number>6</number>
                 </property>
                 <property name="singleStep">
                  <double>0.000001000000000</double>
                 </property>
                 <property name="value">
                  <double>8.280556000000001</double>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnStopRotation">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Stop rotation</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QTabWidget" name="tabWidget_2">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="tabShape">
                <enum>QTabWidget::Rounded</enum>
               </property>
               <property name="currentIndex">
                <number>0</number>
               </property>
               <property name="documentMode">
                <bool>true</bool>
               </property>
               <widget class="QWidget" name="tab_3">
                <attribute name="title">
                 <string>Speed</string>
                </attribute>
                <layout class="QGridLayout" name="gridLayout_6">
                 <item row="0" column="0">
                  <widget class="QListWidget" name="lwdgENC1">
                   <property name="baseSize">
                    <size>
                     <width>0</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="autoFillBackground">
                    <bool>false</bool>
                   </property>
                   <property name="frameShape">
                    <enum>QFrame::NoFrame</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <property name="lineWidth">
                    <number>0</number>
                   </property>
                   <property name="midLineWidth">
                    <number>0</number>
                   </property>
                   <property name="verticalScrollBarPolicy">
                    <enum>Qt::ScrollBarAlwaysOff</enum>
                   </property>
                   <property name="sizeAdjustPolicy">
                    <enum>QAbstractScrollArea::AdjustToContents</enum>
                   </property>
                   <property name="viewMode">
                    <enum>QListView::ListMode</enum>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="tab_4">
                <attribute name="title">
                 <string>aux</string>
                </attribute>
               </widget>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
